jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MojoCommon.java

```diff
@@ -80,6 +80,9 @@ static void finishUpdateChecker(
                       "\u001B[33m"
                           + ProjectInfo.GITHUB_URL
                           + "/blob/master/jib-maven-plugin/CHANGELOG.md\u001B[0m"));
+              projectProperties.log(
```

Comment:
This one has a slightly different format than the one above (two calls to log, rather than a single giant log)? I think it would make sense to have them be consistent?

Reply:
This is consistent; gradle separates lines with `\n`, but maven needs to make separate log calls since each lines starts with `[INFO]`.

Reply:
Cool that makes sense